CONTRIBUTING.md

```diff
@@ -1,22 +1,22 @@
-# Contributing to this standard
+# Contributing to IT-Wallet Technical Documentation
 
 🙇‍♀️ Thank you for contributing!
 
-We – the maintainers and contributors of this project – understand that a standard like this can only be set in collaboration with as many public technologists, policy makers and interested folk as possible. Thus we appreciate your input, enjoy feedback and welcome improvements to this project and are very open to collaboration.
+We – the maintainers and contributors of this project – understand that a Technical Documentation like this can only be set in collaboration with as many public technologists, policy makers and interested folk as possible. Thus we appreciate your input, enjoy feedback and welcome improvements to this project and are very open to collaboration.
```

Comment:
please replace personal language (we, you and so on) with impersonal formal language